src/data/scripts/huggingface_create/README.md

```diff
@@ -0,0 +1,40 @@
+# Hugging Face Dataset Creation Script
+This script is designed to create datasets for use with the Hugging Face Transformers library. It takes JSONL files containing data and converts them into a format compatible with the Hugging Face Datasets library. The resulting dataset can then be easily loaded and used for training and evaluation in various NLP tasks.
+
+## Usage
+This script is intended to be used for creating custom datasets that can be utilized with Hugging Face Transformers. It takes as input a set of JSONL files containing data and produces a Hugging Face dataset in the desired format.
+
+## Dependencies
+Before using this script, make sure you have the following dependencies installed:
+
+datasets>=2.12.0
+pandas>=2.0.2
+tqdm>=4.65.0
+You can typically install these dependencies using pip:
+
+```bash
+pip install datasets>=2.12.0 pandas>=2.0.2 tqdm>=4.65.0
```

Comment:
`>=` means when datasets 3.0.0 is released, this code will have a high chance of breaking. this symbol `X.x.x` , the first X version usually means "breaking changes". "breaking changes" represent library API change that the application code may need modification to be able to have the same functionality.

Reply:
done